### PR TITLE
Added SmartAP PRO bootloader configuration

### PR DESCRIPTION
**Pull request by aviaks**

### CODE REVIEW

File: hw_config.h [hunk 1]
```diff
@@ -970,6 +970,36 @@
...
+ ****************************************************************************/
+
+#elif  defined(TARGET_HW_SMARTAP_PRO)
+
+# define APP_LOAD_ADDRESS               0x08008000
+# define BOOTLOADER_DELAY               5000
+# define BOARD_SMARTAP_PRO
+# define INTERFACE_USB                  1
+# define INTERFACE_USART                0
+# define USBDEVICESTRING                "SmartAP PRO BL"
```
Comment (davids5) [inline hw_config.h]: You will need to add the prefix "PX4 BL" or retool the uploader 

https://github.com/PX4/Bootloader/blob/f7a97e3a776bd386495c4b771e07555ad5688315/hw_config.h#L11-L18

File: hw_config.h [hunk 2]
```diff
@@ -970,6 +970,36 @@
...
 
+/****************************************************************************
+ * TARGET_HW_SMARTAP_PRO
+ ****************************************************************************/
+
+#elif  defined(TARGET_HW_SMARTAP_PRO)
+
+# define APP_LOAD_ADDRESS               0x08008000
+# define BOOTLOADER_DELAY               5000
+# define BOARD_SMARTAP_PRO
```
Comment (davids5) [inline hw_config.h]: delete

File: hw_config.h [hunk 3]
```diff
@@ -970,6 +970,36 @@
...
+# define INTERFACE_USB                  1
+# define INTERFACE_USART                0
+# define USBDEVICESTRING                "SmartAP PRO BL"
+# define USBPRODUCTID                   0x0011
```
Comment (davids5) [inline hw_config.h]: Match the BOARD_TYPE  0x20

**Comment by aviaks** [comment]
Thanks @davids5, requested changes made.

**Comment by davids5** [comment]
@aviaks - I rebased and merged this to master.

**Pull request closed without merging**